state-sync/storage-service/server/src/lib.rs

```diff
@@ -52,6 +55,17 @@ pub enum Error {
     UnexpectedErrorEncountered(String),
 }
 
+impl Error {
+    /// Returns a summary label for the error type
+    fn to_label(&self) -> String {
```

Comment:
Could probably return `&'static str` unless you're looking specifically to have a `String` allocated.